cmake/winml.cmake

```diff
@@ -462,26 +471,26 @@ if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
 endif("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
 
 # Link libraries
-target_link_libraries(winml_dll PRIVATE windowsapp.lib)
-target_link_libraries(winml_dll PRIVATE libprotobuf)
-target_link_libraries(winml_dll PRIVATE onnx)
-target_link_libraries(winml_dll PRIVATE onnxruntime_common)
-target_link_libraries(winml_dll PRIVATE onnxruntime_graph)
-target_link_libraries(winml_dll PRIVATE onnxruntime_framework)
-target_link_libraries(winml_dll PRIVATE onnxruntime_mlas)
-target_link_libraries(winml_dll PRIVATE onnxruntime_optimizer)
-target_link_libraries(winml_dll PRIVATE onnxruntime_providers)
-target_link_libraries(winml_dll PRIVATE onnxruntime_providers_dml)
-target_link_libraries(winml_dll PRIVATE onnxruntime_session)
-target_link_libraries(winml_dll PRIVATE onnxruntime_util)
-target_link_libraries(winml_dll PRIVATE onnx_proto)
+#target_link_libraries(winml_dll PRIVATE libprotobuf)
+#target_link_libraries(winml_dll PRIVATE onnx)
+#target_link_libraries(winml_dll PRIVATE onnxruntime_common)
+#target_link_libraries(winml_dll PRIVATE onnxruntime_graph)
+#target_link_libraries(winml_dll PRIVATE onnxruntime_framework)
+#target_link_libraries(winml_dll PRIVATE onnxruntime_mlas)
+#target_link_libraries(winml_dll PRIVATE onnxruntime_optimizer)
+#target_link_libraries(winml_dll PRIVATE onnxruntime_providers)
+#target_link_libraries(winml_dll PRIVATE onnxruntime_providers_dml)
+#target_link_libraries(winml_dll PRIVATE onnxruntime_session)
+#target_link_libraries(winml_dll PRIVATE onnxruntime_util)
+#target_link_libraries(winml_dll PRIVATE onnx_proto)
+target_link_libraries(winml_dll PRIVATE onnxruntime)
 target_link_libraries(winml_dll PRIVATE re2)
 target_link_libraries(winml_dll PRIVATE wil)
+target_link_libraries(winml_dll PRIVATE windowsapp.lib)
 target_link_libraries(winml_dll PRIVATE winml_lib_api)
-target_link_libraries(winml_dll PRIVATE winml_lib_core)
+#target_link_libraries(winml_dll PRIVATE winml_lib_core)
 target_link_libraries(winml_dll PRIVATE winml_lib_image)
 target_link_libraries(winml_dll PRIVATE winml_lib_telemetry)
-target_link_libraries(winml_dll PRIVATE onecoreuap_apiset.lib)
```

Comment:
>onecoreuap_apiset [](start = 40, length = 17)

intentional? you replaced onecoreuap_apiset.lib with windowsapp.lib

Reply:
yeah, not sure why i did that.   

---
In reply to: [347039801](https://github.com/microsoft/onnxruntime/pull/2406#discussion_r347039801) [](ancestors = 347039801)

Reply:
this doesn't seem resolved yet...

---
In reply to: [347539379](https://github.com/microsoft/onnxruntime/pull/2406#discussion_r347539379) [](ancestors = 347539379,347039801)